content/en/docs/instrumentation/js/libraries.md

```diff
@@ -2,10 +2,47 @@
 title: Using instrumentation libraries
 linkTitle: Libraries
 weight: 40
-spelling: cSpell:ignore autoinstrumentation metapackage
+description: How to instrument libraries an app depends on
+spelling: cSpell:ignore autoinstrumentation metapackage metapackages
 ---
 
-You can use
+When you develop an app, you make use of 3rd party libraries and frameworks, to
+accelerate your work and to not reinvent the wheel. If you now instrument your
+app with OpenTelemetry, you don't want to spend additional time on manually
+adding traces, logs and metrics to those libraries and frameworks. Gladly, you
```

Comment:
```suggestion
adding traces, logs and metrics to those libraries and frameworks. Fortunately, you
```
Not sure about the wording, maybe use fortunately instead of gladly? This paragraph feels a little wordy to me but it can be adjusted later if something comes to mind.